src/main/resources/assets/twilightforest/lang/fr_fr.json

```diff
@@ -108,250 +114,361 @@
   "block.twilightforest.force_field_pink": "Champ de force rose",
   "block.twilightforest.force_field_orange": "Champ de force orange",
   "block.twilightforest.force_field_green": "Champ de force vert",
   "block.twilightforest.force_field_blue": "Champ de force bleu",
+
   "block.twilightforest.ironwood_block": "Bloc de bois de fer",
   "block.twilightforest.fiery_block": "Bloc de métal ardent",
-  "block.twilightforest.steeleaf_block": "Bloc de Stealeaf",
+  "block.twilightforest.steeleaf_block": "Bloc de feuille voleuse",
   "block.twilightforest.arctic_fur_block": "Bloc de fourrure arctique",
+
   "block.twilightforest.carminite_block": "Bloc de carminite",
-  "block.twilightforest.cinder_furnace": "Four à cendres [NYI]",
-  "block.twilightforest.cinder_log": "Bois de cendres [NYI]",
-  //"block.twilightforest.cinder_wood": "Cinder Wood [NYI]",
-  //"block.twilightforest.twilight_portal_miniature_structure": "Miniature Twilight Portal [NYI]",
-  //"block.twilightforest.naga_courtyard_miniature_structure": "Miniature Naga Courtyard [NYI]",
-  //"block.twilightforest.lich_tower_miniature_structure": "Miniature Lich Tower [NYI]",
-  //"block.twilightforest.stone_twist": "Stone Twist",
-  "block.twilightforest.spiral_bricks": "Pierre taillée en spirale",
-  "block.twilightforest.lapis_block": "Bloc de lapis [NYI]",
-  "block.twilightforest.etched_nagastone": "Nâgastone gravée",
-  "block.twilightforest.nagastone_pillar": "Pilier de nâgastone",
-  "block.twilightforest.nagastone_stairs_left": "Escaliers en nâgastone (gauche)",
-  "block.twilightforest.nagastone_stairs_right": "Escaliers en nâgastone (droite)",
-  "block.twilightforest.etched_nagastone_mossy": "Nâgastone gravée moussue",
-  "block.twilightforest.nagastone_pillar_mossy": "Pilier de nâgastone moussue",
-  "block.twilightforest.nagastone_stairs_mossy_left": "Escaliers de nâgastone moussue (gauche)",
-  "block.twilightforest.nagastone_stairs_mossy_right": "Escaliers de nâgastone moussue (droite)",
-  "block.twilightforest.etched_nagastone_weathered": "Nâgastone gravée en ruine",
-  "block.twilightforest.nagastone_pillar_weathered": "Pilier en nâgastone en ruine",
-  "block.twilightforest.nagastone_stairs_weathered_left": "Escaliers en nâgastone altérée (gauche)",
-  "block.twilightforest.nagastone_stairs_weathered_right": "Escaliers en nâgastone altérée (droite)",
+  "block.twilightforest.cinder_furnace": "Four à cendres",
+
+  "block.twilightforest.cinder_log": "Bûche de cendres",
+  "block.twilightforest.cinder_wood": "Bois de cendres",
+
+  "block.twilightforest.twilight_portal_miniature_structure": "Portail Twilight miniature",
+  "block.twilightforest.naga_courtyard_miniature_structure": "Cour miniature du Naga",
+  "block.twilightforest.lich_tower_miniature_structure": "Tour miniature de la Lich",
+
+  "block.twilightforest.stone_twist": "Pierre torsadée",
+  "block.twilightforest.stone_twist_thin": "Twisted Stone Pillar",
```

Comment:
looks like a few translations were missed here

Reply:
what translations are missing? I've added some of them but they might be not in the same order as us_US

Reply:
starting at twisted stone pillar down to death tome spawner. I assumed these werent translated